main.go

```diff
@@ -17,20 +17,25 @@ import (
 	"github.com/BurntSushi/toml"
 )
 
+var (
+	regexpNumericalURLVars = regexp.MustCompile(`/[0-9]+[/\s]`)
```

Comment:
I guess following regexp might be useful to normalize the numerical field of query parameter

```
egexpNumericalURLVars = regexp.MustCompile(`[/=][0-9]+[/\s&]`)
```

But I haven't check it yet.

Reply:
I've checked this pattern

```
   21  25.158  1.198000  0.000000  1.198  1.198  1.198  1.198  1.198  1.198   21    0    0    0       51849       2469       2469       2469  GET /api/users/<num>?foo=<num>&bar=<num> HTTP/1.1
```